test/test_files/copy/copy_node.test

```diff
@@ -0,0 +1,48 @@
+-NAME Subset
+-QUERY MATCH (row:tableTypes) WHERE row.id >= 20 AND row.id <= 24 RETURN *;
+---- 5
+20|0|57.579280|True|1731-09-26|1731-09-26 03:30:08|OdM
+21|7|64.630960|False|1307-01-26|1307-01-26 03:31:08|AjbxHQThEtDDlOjbzMjCQSXlvGQEjcFLykESrnFHwPKX
+22|71|37.963386|True|1455-07-26|1455-07-26 03:07:03|dRvHHdyNXYfSUcicaxBoQEKQUfgex
+23|58|42.774957|False|1181-10-16|1181-10-16 18:19:43|ISImRVpUjynGMFRQyYmeIUVjM
+24|75|53.813224|False|1942-10-24|1942-10-24 09:30:16|naDlQ
+
+-NAME CheckLines
+-QUERY MATCH (row:tableTypes) RETURN count(*)
+---- 1
+49999
+
+-NAME CountTrue
+-QUERY MATCH (row:tableTypes) WHERE row.column3 = True RETURN count(*)
+---- 1
+24997
+
+-NAME countFalse
```

Comment:
Be consistent with your test names. countFalse vs CountTrue. Let's stick with always capitalizing.